src/pages/ViewProject.tsx

```
@@ -0,0 +1,419 @@
+
+  if (!project) {
+    return (
+      <Box sx={{ textAlign: "center", mt: 6 }}>
+        <Typography color="error" variant="h5">
+          Project not found.
+        </Typography>
+      </Box>
+    );
+  }
+  // Prepare media array combining images and videos
+  const media = useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)